backend/tracim_backend/lib/core/event.py

```diff
@@ -106,17 +106,20 @@ def _filter_event_types(
         if event_types:
             event_type_filters = []
             for event_type in event_types:
-                if event_type.subtype:
-                    event_type_filter = and_(
-                        Event.entity_type == event_type.entity,
-                        Event.operation == event_type.operation,
-                        Event.entity_subtype == event_type.subtype,
-                    )
+                if event_type.operation:
```

Comment:
reaction are not the first event to not have subtype, does this mean we do not use this code for other event (like user, workspace, ...) ?

Reply:
This code allows excluding all the events related to a given entity.  It was not possible before.

Reply:
it's ok.